### DISCUSSION TITLE
"default branch" is used for two different things.

### DISCUSSION THREAD

**Discussion by jvanlangen** [community/community#57007] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

Hi,

Imagine having a **main** branch and a **dev** branch. When pull requests are made, i'd like to have them in de **dev** branch. We release all these features and bugfixes ones a month.
Now our projects are forks from the main.

The problem is, when i set the "default branch" to **main**, all my pull requests are going default to the **main**. I'd rather have them in de **dev**. If I change the "default branch" to **dev**. The pull request are default selecting the **dev** branch. The problem then is that when I want to create a fork, it is only possible from the dev.

I'd rather see two different settings for this:

- default PR branch
- default Fork branch

**Comment by hades200082** [comment]
This has been a requested feature since 2018 - https://github.com/isaacs/github/issues/1401
